tests/schema/helpers.js

```diff
@@ -128,6 +131,14 @@ export function tryTest(nth, max, test, refs, schema, callback) {
       expect(sample).to.eql(test.equal);
     }
 
+    if (test.throws) {
```

Comment:
Forcing tests to fail if they were supposed to throw. There are 7 skipped tests that are now failing. I will investigate them in https://github.com/json-schema-faker/json-schema-faker/issues/579.